circle/circle.go

```diff
@@ -17,6 +18,13 @@ Square: 201.06192982974676
 */
 func main() {
 	fmt.Println("The program prints the perimeter and the square of a circle given the radius.")
+	fmt.Println("enter radius")
+	var r float64
+	fmt.Scanln(&r)
+	p := r * 2 * math.Pi
```

Comment:
nit: in English it's usually the area, and not the square. It doesn't affect the correctness of your solution